flutter_local_notifications/darwin/Classes/Converters.h

```diff
@@ -10,8 +10,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface Converters : NSObject
 
-+ (UNNotificationCategoryOptions)parseNotificationCategoryOptions:(NSArray *)options API_AVAILABLE(ios(10.0));
-+ (UNNotificationActionOptions)parseNotificationActionOptions:(NSArray *)options API_AVAILABLE(ios(10.0));
++ (UNNotificationCategoryOptions)parseNotificationCategoryOptions:
```

Comment:
Do correct me I'm wrong but as this is being shared, won't some tweaks need to be done check if there's a compatibile macOS version too? i.e. it'd become `API_AVAILABLE(ios(10), macosx(10.14))`

Reply:
@ened example reference https://github.com/FirebaseExtended/flutterfire/blob/e3ff56d540db0a5c7511be5b1dfd8c0d1b86d192/packages/firebase_messaging/firebase_messaging/ios/Classes/FLTFirebaseMessagingPlugin.m#L528

Reply:
OK!